crates/jstz_node/src/services/logs/mod.rs

```diff
@@ -258,6 +204,26 @@ impl LogsService {
     }
 }
 
+#[derive(Deserialize, Debug)]
+#[cfg_attr(not(feature = "persistent-logging"), allow(dead_code))]
+pub struct Pagination {
```

Comment:
nit: 
could implement default here given that it's only used for persistent-logging?

Reply:
Done